magprime/templates/static_views/star_rod_wand.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+  <title>Star Rod Wand</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Star Rod Wand</h1>
+
+<p>
+  What kind of a SUPER STAR would be caught off guard without their trusty
+  star rod wand to defend themselves?
+</p>
+<p>
+  This custom-made plastic molded stick of fabulousness is manufactured
+  exclusively for MAGFest 2018, and is available exclusively for purchase by
```

Comment:
Hm... this text implies that purchasing this tier gives you the _ability_ to purchase the star rod wand, but I'm gonna assume that our 'invitation' tier is the only tier like that. Suggested rewording: "available exclusively through the Super Star badge upgrade." OR just copy and paste the wording from the star umbrella page.